src/utils/theme/index.js

```diff
@@ -1,6 +1,7 @@
 export const COLORS = {
+  almostBlack: '#16202A',
   alto: '#E0E0E0',
-  black: '#1E1F20',
+  black: '#FFFFFF', // changed to white, before is #1E1F20
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color named `black` is set to white (`#FFFFFF`). This could lead to confusion in the future. Consider renaming this color to something more descriptive of its actual color.
```diff
-  black: '#FFFFFF', // changed to white, before is #1E1F20
+  white: '#FFFFFF',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->